vi is a much better "default" editor for void, as it is shipped in base-system,
unlike nano

--- a/rpi-eeprom-config
+++ b/rpi-eeprom-config
@@ -184,8 +184,8 @@
     """
     Implements something like 'git commit' for editing EEPROM configs.
     """
-    # Default to nano if $EDITOR is not defined.
-    editor = 'nano'
+    # Default to vi if $EDITOR is not defined.
+    editor = 'vi'
     if 'EDITOR' in os.environ:
         editor = os.environ['EDITOR']
 
@@ -515,7 +515,7 @@
 
    To cancel the pending update run 'sudo rpi-eeprom-update -r'
 
-   The default text editor is nano and may be overridden by setting the 'EDITOR'
+   The default text editor is vi and may be overridden by setting the 'EDITOR'
    environment variable and passing '-E' to 'sudo' to preserve the environment.
 
 6. Signing the bootloader config file.
